RevenueCatUI/Templates/V2/Components/Text/TextComponentViewModel.swift

```diff
@@ -157,7 +157,7 @@ struct TextComponentStyle {
     let visible: Bool
     let text: String
     let fontWeight: Font.Weight
-    let color: Color
+    let color: PaywallComponent.ColorScheme
```

Comment:
Storing the `ColorScheme` instead of the dynamic color now because it can potentially be a gradient.